package-lock.json

```diff
@@ -1131,6 +1131,7 @@
       "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
       "integrity": "sha512-5hYdAkZlcG8tOLujVDTgCT+uPX0VnpAH28gWsLfzpXYm7wP6mp5Q/gYyR7YQ0cKVJcXJnl3j2kpBan13PtQf6w==",
       "dev": true,
+      "optional": true,
```

Comment:
Est-ce que tu as `pull --rebase` avant de faire un pull request ? Parce qu'il y a des changements surtout au niveau des packages que je suis pas sûr que c'est ce que tu voulais faire

Reply:
My bad our branch wasn't up to date with that of the solutionspace